.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ramls/raml-util"]
+	path = ramls/raml-util
+	url = https://github.com/folio-org/raml
```

Comment:
Please, specify branch of raml-util submodule
```
branch = raml1.0

```
And add definition of acq-models:
```
[submodule "ramls/acq-models"]
	path = ramls/acq-models
	url = https://github.com/folio-org/acq-models.git
```